base/shared/src/main/scala/scalaz/data/maybe2.scala

```diff
@@ -42,31 +42,39 @@ final class Just2Extractor[A, B] private[data] (private val value: Option2[A, B]
   def _2: B                     = value._2
 }
 
+object Maybe2Module {
+  implicit def isCovariant_1[B]: IsCovariant[Maybe2[?, B]] = Maybe2.isCovariant_1
+  implicit def isCovariant_2[A]: IsCovariant[Maybe2[A, ?]] = Maybe2.isCovariant_2
+  implicit def eq[A, B](implicit A: Eq[A], B: Eq[B]): Eq[Maybe2[A, B]] =
+    Maybe2.eq[A, B]
+  implicit def bifunctor: Bifunctor[Maybe2] = Maybe2.bifunctor
```

Comment:
`Traversable` etc?

Reply:
I just exposed the instances that were not being exposed properly; adding new ones is for later.